.nowignore

```diff
@@ -1,4 +1,18 @@
-/src
-/tests
-!/src/sentry/static/sentry
-!/src/sentry/locale
+*
```

Comment:
What's this do?

Reply:
Ignore everything and then we add things.

I couldn't get it working the other way around, though I kind of like this as it makes builds way faster since it doesn't have to download all of our source code.